packages/patternfly-3/patternfly-react/less/pagination.less

```diff
@@ -0,0 +1,6 @@
+.pagination > li.disabled {
+  cursor: not-allowed;
```

Comment:
I'm not sure of the need to change the cursor to not-allowed.  I don't think we do this in other areas, although it does make it clear.  @jeff-phillips-18 what do you think ?

Reply:
https://rawgit.com/patternfly/patternfly-react/gh-pages/patternfly-3/index.html?knob-View%20Type%3A=list&knob-Page=1&knob-Number%20of%20Pages=5&knob-Page%20Size%20Drop%20Up=true&knob-Item%20Count%3A=75&knob-Items%20Start%3A=1&knob-Items%20End=15&selectedKind=patternfly-react%2FWidgets%2FPagination&selectedStory=Pagination%20row&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

Reply:
I actually agree on the cursor. I added this line because i saw the same behaviour in the current [storybook](https://rawgit.com/patternfly/patternfly-react/gh-pages/patternfly-3/index.html?knob-View%20Type%3A=list&knob-Page=1&knob-Number%20of%20Pages=5&knob-Page%20Size%20Drop%20Up=true&knob-Item%20Count%3A=75&knob-Items%20Start%3A=1&knob-Items%20End=15&selectedKind=patternfly-react%2FWidgets%2FPagination&selectedStory=Pagination%20row&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs)

Reply:
I see the cursor already being set to `not-allowed` on the `<a>` so I don't think we need it on the `<li>` regardless.

https://github.com/patternfly/patternfly/blob/master/src/less/pagination.less#L46